File: LeetCode/191. Number of 1 Bits/Solution.cs
public class Solution
{
    public int HammingWeight(uint n)
    {
        var result = Convert.ToString(n, 2);

        //for(var i = 0 ; i < 32; i++) result+= (n & (1 << i)) != 0 ? 1 : 0;

        return result.Count(bit => bit == '1');
    }
}

File: LeetCode/191. Number of 1 Bits/Solution.js
/**
 * @param {number} n - a positive integer
 * @return {number}
 */
var hammingWeight = function (n) {
  var count = 0;

  while (n !== 0) {
    count += n & 1;

    n >>>= 1;
  }

  return count;
};
